style/app.scss

```diff
@@ -2487,14 +2480,44 @@ main-filter input
 
 .tooltip-group-display
 {
-  cursor: pointer;
-  transition: opacity 0.4s linear;
-  opacity:1;
+  cursor     : pointer;
```

Comment:
Revert the style changes for this rule and the one below. You can make these in a separate PR.